Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -5718,13 +5723,21 @@ def apply_security_profile(pre_post: str, rule_name: str, profile_type: str, pro
 
 def apply_security_profile_command(profile_name: str, profile_type: str, rule_name: str, pre_post: str = None):
 
-    if DEVICE_GROUP:
+    if DEVICE_GROUP:  # Panorama instance
         if not pre_post:
             raise Exception('Please provide the pre_post argument when applying profiles to rules in '
                             'Panorama instance.')
+        panorama_xpath = f"{XPATH_RULEBASE}{pre_post}/security/rules/entry[@name='{rule_name}']/profile-setting/"\
+                         f"profiles/{profile_type}",
+        apply_security_profile(panorama_xpath, profile_name)
+        return_results(f'The profile {profile_name} has been applied to the rule {rule_name}')
+
+    else:  # firewall instance
+        firewall_xpath = f"{XPATH_RULEBASE}rulebase/security/rules/entry[@name='{rule_name}']/profile-setting/"\
+                         f"profiles/{profile_type}"
```

Comment:
are you sure that in the firewall case the `rulebase` is constant?
```suggestion
        xpath = f"{XPATH_RULEBASE}{pre_post}/security/rules/entry[@name='{rule_name}']/profile-setting/"\
                f"profiles/{profile_type}"

    else:  # firewall instance
        xpath = f"{XPATH_RULEBASE}rulebase/security/rules/entry[@name='{rule_name}']/profile-setting/"\
                f"profiles/{profile_type}"
```